src/courses/views.py

```diff
@@ -261,6 +261,22 @@ class BlockDelete(NonPublicOnlyViewMixin, DeleteView):
     model = Block
     success_url = reverse_lazy('courses:block_list')
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        course_student_qs = self.get_object().coursestudent_set
+        context["population"] = course_student_qs.count()
+        context['populated'] = course_student_qs.exists()
```

Comment:
The mixture of single and double quotes is triggering me.  We don't actually have a standard in this project which to use when, but side by side like this means I can't even pretend that we do, lol.

Reply:
fixed